src/sass/components/_tour.scss

```diff
@@ -29,3 +29,20 @@
   }
 }
 /* stylelint-enable declaration-no-important */
+
+// Override background image
+.slds-popover_walkthrough .slds-popover__header {
+  background-image: url('~img/popover-header.png');
+}
+
+.popover-trigger {
+  color: $palette-violet-65;
```

Comment:
@dvdherron We're using SLDS variables for most things: https://www.lightningdesignsystem.com/design-tokens/

@SondraE The color used in the mockup doesn't seem to come from the SLDS color palette; is that correct? If so, could we substitute this `$palette-violet-65` (`#d17dfe`), or do you have another one you prefer from that list?